test/controllers/fhir/r4/api_controller_test.rb

```diff
@@ -831,33 +831,54 @@ def setup_patients
+    state_iss = issues.find { |e| /Old York.*State/.match(e['diagnostics']) }
+    assert(FHIRPath.evaluate(state_iss['expression'].first, json_patient) == 'Old York')
+    eth_iss = issues.find { |e| /0000.*Ethnicity/.match(e['diagnostics']) }
+    assert_not_nil eth_iss
+    # FHIRPath lib errors on paths with nested extensions. The below assertion should work.
+    # assert(FHIRPath.evaluate(eth_iss['expression'].first, json_patient) == '0000')
+    pct_iss = issues.find { |e| /High noon.*Preferred Contact Time/.match(e['diagnostics']) }
+    assert(FHIRPath.evaluate(pct_iss['expression'].first, json_patient) == 'High noon')
+    sex_iss = issues.find { |e| /On FHIR.*Sex/.match(e['diagnostics']) }
+    assert(FHIRPath.evaluate(sex_iss['expression'].first, json_patient) == 'On FHIR')
+    phone_type_iss = issues.find { |e| /Dumbphone.*Telephone Type/.match(e['diagnostics']) }
+    assert_not_nil phone_type_iss
+    # FHIRPath lib errors on paths with nested extensions. The below assertion should work.
+    # assert(FHIRPath.evaluate(phone_type_iss['expression'].first, json_patient) == 'Dumbphone')
+    phone_iss = issues.find { |e| /123.*Primary Telephone/.match(e['diagnostics']) }
+    assert(FHIRPath.evaluate(phone_iss['expression'].first, json_patient) == '123')
+    dob_iss = issues.find { |e| /Date of Birth/.match(e['diagnostics']) }
+    assert(FHIRPath.evaluate(dob_iss['expression'].first, json_patient) == '2000')
+    lde_iss = issues.find { |e| /Last Date of Exposure/.match(e['diagnostics']) }
+    assert_not_nil lde_iss # LDE is omitted from the request, so don't eval the FHIRPath
+    sod_iss = issues.find { |e| /1492.*Symptom Onset/.match(e['diagnostics']) }
+    assert(FHIRPath.evaluate(sod_iss['expression'].first, json_patient) == '1492')
+    add_travel_start_date_iss = issues.find { |e| /1776.*Additional Planned Travel Start Date/.match(e['diagnostics']) }
+    assert(FHIRPath.evaluate(add_travel_start_date_iss['expression'].first, json_patient) == '1776')
+    dod_iss = issues.find { |e| /2020-01-32.*Date of Departure/.match(e['diagnostics']) }
+    assert(FHIRPath.evaluate(dod_iss['expression'].first, json_patient) == '2020-01-32')
+    doa_iss = issues.find { |e| /9999-99-99.*Date of Arrival/.match(e['diagnostics']) }
+    assert(FHIRPath.evaluate(doa_iss['expression'].first, json_patient) == '9999-99-99')
+    last_iss = issues.find { |e| /Last Name/.match(e['diagnostics']) }
+    assert_not_nil last_iss # Last name is omitted from the request, so don't eval the FHIRPath
+    assigned_usr_iss = issues.find { |e| /10000.*Assigned User/.match(e['diagnostics']) }
+    assert(FHIRPath.evaluate(assigned_usr_iss['expression'].first, json_patient) == 1_000_000)
+    email_iss = issues.find { |e| /Email.*Primary Contact Method/.match(e['diagnostics']) }
+    assert_not_nil email_iss # Last name is omitted from the request, so don't eval the FHIRPath
```

Comment:
Nit-pick here: Copy paste error on the comment, same on line 1295 too.

Reply:
Fixed https://github.com/SaraAlert/SaraAlert/pull/743/commits/7c86dcbe6481df1a8ebebd027c6b6e9a819845b1